extensions/capability-negotiation.md

```diff
@@ -131,6 +131,14 @@ with several subcommands.  The command added is named `CAP`.  `CAP` takes a sing
 required subcommand, optionally followed by a single parameter. Each subcommand defines any
 further parameters.
 
+The format of a client-sent `CAP` message is:
+
+    CAP <subcommand> [<param>{,<param>}]
```

Comment:
Hm, I find the `[<param>{,<param>}]` a bit confusing, is it used anywhere else? (`{` and `,` in particular)

Reply:
Taken from https://modern.ircdocs.horse/#join-message

Reply:
Ah, that one is different, because the JOIN messages accepts two comma-separated arguments:

    JOIN #chan1,#chan2 key1,key2

Reply:
My mistake, will edit.